### ISSUE TITLE
problem with Keras

### ISSUE THREAD

**Issue by marekjaszuk** [jianing-sun/Mask-YOLO#3]
Hi,
I,m trying to run the infer_shapes.py, and I get the below error from the keras_applications
I don't know how to deal with this. The obvious solutions suggested by the error message do not change anything. Did anyone have such an issue?
I have Tensorflow 1.12.0 and Keras 2.2.4.

The error message:
Using TensorFlow backend.
Traceback (most recent call last):
  File "infer_shapes.py", line 8, in <module>
    import myolo.model as modellib
  File "../../myolo/model.py", line 35, in <module>
    backend = get_keras_submodule('backend')
  File "/usr/local/lib/python3.5/dist-packages/keras_applications/__init__.py", line 37, in get_keras_submodule
    raise ImportError('You need to first `import keras` '
ImportError: You need to first `import keras` in order to use `keras_applications`. For instance, you can do:

```
import keras
from keras_applications import vgg16
```

Or, preferably, this equivalent formulation:

```
from keras import applications
```

**Comment by jianing-sun** [comment]
seems like it's the problem of the backend config? can you try [this](https://stackoverflow.com/questions/51772437/importerror-you-need-to-first-import-keras-in-order-to-use-keras-application)?

**Comment by marekjaszuk** [comment]
Yes, I've read this before I posted this issue, but this didn't give me the solution. Finally I've found how to solve it. The problem was, that the _KERAS_BACKEND variable in /usr/local/lib/python3.5/dist-packages/keras_applications/init.py was not initialized. So in model.py I called set_keras_submodules(K.backend()), which initialized the variable with tensorflow. So the program passed this issue, but I got stucked on the next issue:
File "../../myolo/model.py", line 74, in mobilenet_graph
    x = _depthwise_conv_block(x, 64, alpha, depth_multiplier, block_id=1)
  File "/usr/local/lib/python3.5/dist-packages/keras_applications/mobilenet.py", line 442, in _depthwise_conv_block
    channel_axis = 1 if backend.image_data_format() == 'channels_first' else -1
AttributeError: 'NoneType' object has no attribute 'image_data_format'

I've found, that again there seems to be a problem with the backend variable initialization, which is in mobilenet.py. But I haven't found yet how to initialize it. 
I don't know why so many problems do I have with running Your program. Which version of Tensorflow, and Keras did you use to run it? Maybe there is a version mismatch.
